### DISCUSSION TITLE
Cetak Kwitansi, Input No. Referensi, dan Upload Foto Kwitansi

### DISCUSSION THREAD

**Discussion by nafiesl** [buku-masjid/buku-masjid#44] [Ideas]
## Goal

Ada ide dari salah satu user, agar dibuatkan fitur kwitansi pada setiap transaksi, agar lebih rinci dalam tracking riwayat transaksi. Kemungkinan fitur yang diperlukan:

- Input No. Ref (nomor referensi) pada setiap input transaksi
- Tombol cetak kwitansi pada setiap transaksi
- Upload foto kwitansi / bukti transaksi

**Comment by nafiesl** [comment]
Kemungkinan implementasi:
1. Tiap transaksi dibuatkan halaman detail detail transaksi
2. User disediakan tombol untuk cetak kuitansi (PDF)
3. User bisa upload bukti transaksi di halaman ini, berupa, gambar atau PDF.

**Reply by mharisudn** [reply]
Bismillah, ana coba implementasikan untuk ini mas. kebetulan di pesantren kami juga membutuhkan fitur ini.

**Reply by nafiesl** [reply]
Halaman detail transaksi dan tombol cetak kwitansi sudah selesai di PR ini:
- https://github.com/buku-masjid/buku-masjid/pull/96
